src/pages/studyView/chartHeader/ChartHeader.tsx

```diff
@@ -249,7 +249,8 @@ export class ChartHeader extends React.Component<IChartHeaderProps, {}> {
         const items = [];
         if (
             this.props.chartControls &&
-            !!this.props.chartControls.showLogScaleToggle
+            this.props.chartControls.showLogScaleToggle &&
+            this.props.toggleLogScale
```

Comment:
Why did you make this change? Not saying it's wrong, just doesn't seem to fit the PR

Reply:
I made this change to follow the check I made for the NA toggle, and removed the `!!` since in `ChartContainer.tsx` I changed lines 114-115 so those Boolean checks are now required 
```js
    onToggleLogScale: (chartMeta: ChartMeta) => void;
    onToggleNAValue: (chartMeta: ChartMeta) => void;
```

Reply:
But then again, `ChartControls` has optional parameters, so maybe keeping the corresponding input props optional is better..